body-and-society.csl

```diff
@@ -100,8 +100,8 @@
           </if>
           <else-if variable="URL">
             <text value="URL"/>
```

Comment:
while you're at it, why not remove the two groups with a single item, wrap everything from line 102 to 111 into a group with delimiter=" " and remove all spaces in affixes. Does that make sense?